.github/workflows/aws-deploy.yml

```diff
@@ -90,6 +90,37 @@ jobs:
           cdk_subcommand: 'deploy'
           cdk_args: '--require-approval never --context env=staging --progress events --debug'
           actions_comment: false
+
+  cdk-deploy-dev-refactor:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/dev' }}
+    needs: tests
+    permissions:
+      id-token: write
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+        # Authenticate to AWS using GitHub OIDC
+      - name: Assume AWS Role
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-region: us-east-1
+          role-to-assume: arn:aws:iam::631692904429:role/sagebase-github-oidc-sage-bionetworks-schematic-infra
+          role-session-name: GitHubActions-${{ github.repository_owner }}-${{ github.event.repository.name }}-${{ github.run_id }}
+          role-duration-seconds: 1200
+
+      - name: cdk deploy to org-sagebase-dnt-dev (631692904429)
+        uses: youyo/aws-cdk-github-actions@v2
+        with:
+          cdk_subcommand: 'deploy'
+          cdk_args: '--require-approval never --context env=dev --progress events --debug'
```

Comment:
`env` should probably reference `dev-refactor`?

Reply:
Yes. Good catch. Thank you